src/test/java/ca/uhn/fhir/jpa/starter/ExampleServerR5IT.java

```diff
@@ -4,6 +4,15 @@
 import static org.awaitility.Awaitility.await;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
+import ca.uhn.fhir.context.FhirContext;
```

Comment:
I don't love this re-ordering of imports. Can these changes be removed from this commit?